helm/botkube/templates/clusterrole.yaml

```diff
@@ -10,23 +10,26 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 rules:
 {{- if .Values.podSecurityPolicy.enabled }}
-  - apiGroups: ["extensions"]
-    resourceNames: ["botkube-psp"]
-    resources: ["podsecuritypolicies"]
-    verbs: ["use"]
+  - apiGroups: [ "extensions" ]
+    resourceNames: [ "botkube-psp" ]
+    resources: [ "podsecuritypolicies" ]
+    verbs: [ "use" ]
 {{ end }}
-  - apiGroups: [""]
-    resources: ["namespaces"]
-    verbs: ["get"]
-  - apiGroups: [""]
-    resources: ["nodes"]
-    verbs: ["list"]
+  - apiGroups: [ "" ]
+    resources: [ "namespaces" ]
+    verbs: [ "get" ]
+  - apiGroups: [ "" ]
+    resources: [ "secrets", "configmaps" ]
+    verbs: [ "get", "list", "watch" ]
+  - apiGroups: [ "" ]
+    resources: [ "nodes" ]
+    verbs: [ "list" ]
   - apiGroups:
-    - ""
+      - ""
```

Comment:
For consistency I'd stick to `[ "" ]`, `[ "secrets", "configmaps" ]` syntax till the end of the file.